bndtools.core/src/bndtools/internal/BndAdapter.java

```diff
@@ -0,0 +1,42 @@
+package bndtools.internal;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.IAdapterFactory;
+import org.osgi.service.component.annotations.Component;
+
+import aQute.bnd.build.Project;
+import bndtools.Plugin;
+import bndtools.central.Central;
+
+@Component(service = IAdapterFactory.class, property = {
```

Comment:
You don't need the `service=` parameter here. If your class implements exactly one interface, the Bnd tooling will automatically use that interface as the service type.

Reply:
I personally better like to be explicit but wont consist on it of course.